api/v1alpha1/common.go

```diff
@@ -87,6 +88,10 @@ type ServiceCommonSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
 	// Service integrations to specify when creating a service. Not applied after initial service creation
 	ServiceIntegrations []*ServiceIntegrationItem `json:"serviceIntegrations,omitempty"`
+
+	// +kubebuilder:validation:MaxItems=10
+	// Technical contact emails of the service
+	TechnicalEmails *[]avnClient.ContactEmail `json:"tech_emails,omitempty"`
```

Comment:
Hey. Please don't use client structs here. We need more control on that fields. Like adding validation rules.

Reply:
Added test in https://github.com/aiven/aiven-operator/pull/576/commits/f94a58472144580ed311e3cc6af0df71beb28689